pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
 addopts = --cov . --pep8 --pylint --ds=edx_sga.test_settings
-norecursedirs = node_modules .git .tox edx_sga.static edx_sga.locale edx_sga.templates {arch} *.egg
+norecursedirs = __pycache__ build node_modules .git .tox edx_sga.static edx_sga.locale edx_sga.templates {arch} *.egg
```

Comment:
SGA dont have 
- node_modules

can you add `edx_sga.egg-info`, `.cache`

Reply:
what is `__pycache__ build`?

Reply:
I had these directories locally but I'm not seeing them anymore, I'll remove them from the list